### ISSUE TITLE
How to add multiple directories in dependabot.yml config file?

### ISSUE THREAD

**Issue by Zethson** [dependabot/dependabot-core#2824]
```
version: 2
updates:
  - package-ecosystem: "composer"
    # Files stored in repository root
    directory: "/"
    schedule:
      interval: "daily"
```

If I want Dependabot to check both, the `/` directory and e.g. `/app` directory for **the same ecosystem**, how would I do that?
Everything that I find online refers to the old UI and not the config file.

**Comment by jamesmcguirepro** [comment]
I have the same problem.

I have multiple projects in different folders and would like a way to list multiple directories for a single package ecosystem, so dependabot can handle all projects at once.

Ie:
```
/Project1/project1.csproj
/Project2/project2.csproj
/Project3/project3.csproj
```

The only way that I have found so far is to create an ecosystem for each folder, which is tedious and repetitive, especially since they all have the same update schedule, default reviewers, default labels and the same list of ignored dependencies.

**Comment by feelepxyz** [comment]
@Zethson you'll have to add each directory like so:

```
version: 2
updates:
  - package-ecosystem: "composer"
    directory: "/"
    schedule:
      interval: "daily"
  - package-ecosystem: "composer"
    directory: "/app"
    schedule:
      interval: "daily"
```

We have some ideas to improve this allowing globs/arrays of directories for the same ecosystem.

**Comment by Zethson** [comment]
@feelepxyz thank you very much.
Not ideal yet, but certainly something to work with.

**Comment by jamesmcguirepro** [comment]
@feelepxyz Is there an issue we can follow to track those ideas for allowing globs/arrays of directories for the same ecosystem?

**Comment by feelepxyz** [comment]
@jamesmcguirepro yup https://github.com/dependabot/dependabot-core/issues/2178

**Comment by Snailedlt** [comment]
Would be neat if directory could take a list or an array of paths too... 
maybe something like this?
```yml
  - package-ecosystem: "composer"
    directory: 
      - "/app"
      - "/"
    schedule:
      interval: "daily"
```

**Comment by jessehouwing** [comment]
Or the ability to use wildcards.
Or that dependabot would recursively search for my manifests.

**Comment by konstantinwirz** [comment]
> for each folder, which is te

any news on this topic?

**Comment by rdhar** [comment]
This can be closed as a duplicate of the older #2178 which has greater visibility.
